62_CAD/fragment_shader.hlsl

```diff
@@ -372,6 +372,21 @@ float32_t4 calculateFinalColor<true>(const uint2 fragCoord, const float localAlp
     return col;
 }
 
+// TODO: place this MSDF code elsewhere
+float median(float r, float g, float b) {
+    return max(min(r, g), min(max(r, g), b));
+}
+
+float screenPxRange(float2 screenSpaceSizePixels, float2 distanceFieldSizePixels) {
+    float2 unitRange = 1.0 / distanceFieldSizePixels;
+    return max(0.5 * dot(unitRange, screenSpaceSizePixels), 1.0);
+}
+
+float msdfDistance(float3 msd, float2 screenPxRangeValue) {
```

Comment:
`screenPxRangeValue` should be `float` not `float2`

Reply:
Done